rclcpp/include/rclcpp/rclcpp.hpp

```diff
@@ -92,9 +92,11 @@
  *   - rclcpp::Event
  * - List topic names and types:
  *   - rclcpp::Node::get_topic_names_and_types()
- * - Get the number of publishers or subscribers on a topic:
+ * - Get the number of publishers, subscribers, clients, or services on a topic:
```

Comment:
```suggestion
 * - Get the number of publishers, subscribers, clients, or services on a topic or service:
```

this is a nitpick, but the comment seems outdated now

Reply:
:smile: :laughing: